cloudformation.json

```diff
@@ -0,0 +1,284 @@
+        }
+      }
+    },
+    "AutoscalingGroup": {
+      "Type": "AWS::AutoScaling::AutoScalingGroup",
+      "Properties": {
+        "LoadBalancerNames": [
+          {
+            "Ref": "LoadBalancer"
+          }
+        ],
+        "VPCZoneIdentifier": {
+          "Ref": "Subnets"
+        },
+        "AvailabilityZones": {
+          "Fn::GetAZs": ""
+        },
+        "LaunchConfigurationName": {
+          "Ref": "LaunchConfig"
+        },
+        "MinSize": "1",
+        "MaxSize": "2",
+        "DesiredCapacity": "1",
+        "Tags": [
+          {
+            "Key": "Stack",
+            "Value": "content-api-first-responder",
+            "PropagateAtLaunch": "true"
+          },
+          {
+            "Key": "Stage",
+            "Value": {
+              "Ref": "Stage"
+            },
+            "PropagateAtLaunch": "true"
+          },
+          {
+            "Key": "App",
+            "Value": "content-api-first-responder",
+            "PropagateAtLaunch": "true"
+          }
+        ]
+      }
+    },
+    "LaunchConfig": {
+      "Type": "AWS::AutoScaling::LaunchConfiguration",
+      "Properties": {
+        "ImageId": "ami-2ce4465f",
+        "InstanceType": "t2.micro",
+        "IamInstanceProfile": { "Ref": "InstanceProfile" },
+        "SecurityGroups": [
+          {
+            "Ref": "FirstResponderSecurityGroup"
+          }
+        ],
+        "AssociatePublicIpAddress": true,
+        "UserData": {
+          "Fn::Base64": {
+            "Fn::Join": [
+              "\n",
+              [
+                "#!/usr/bin/env bash",
+                "wget -NP /home/ubuntu/.ssh --no-check-certificate https://s3-eu-west-1.amazonaws.com/content-api-dist/authorized_keys",
+                "adduser --disabled-password content-api",
+                "cd /home/content-api",
+                "mkdir logs",
+                "mkdir -p /etc/gu",
+                { "Fn::Join": [ "", [ "wget https://s3-eu-west-1.amazonaws.com/content-api-dist/content-api-first-responder/",{ "Ref": "Stage" }, "/content-api-first-responder/content-api-first-responder-1.0-SNAPSHOT.tgz" ] ] },
+                { "Fn::Join": [ "", [ "wget https://s3-eu-west-1.amazonaws.com/content-api-dist/content-api-first-responder/",{ "Ref": "Stage" }, "/content-api-first-responder/content-api-first-responder.service" ] ] },
+                "tar -xvf content-api-first-responder-1.0-SNAPSHOT.tgz",
+                "mv content-api-first-responder-1.0-SNAPSHOT content-api-first-responder",
+                "mv content-api-first-responder.service /etc/systemd/system",
+                { "Fn::Join": [ "", [ "aws s3 cp s3://content-api-config/content-api-first-responder/",{ "Ref": "Stage" }, "/content-api-first-responder.conf /etc/gu/content-api-first-responder.conf" ] ] },
+                "chown -R content-api:content-api /home/content-api /etc/gu",
+                "systemctl start content-api-first-responder.service"
```

Comment:
You are referencing a systemd file here which does not exist. You need to update the existing upstart script to systemd and rename it to match 'content-api-first-responder.service'.